flow/project.py

```diff
@@ -2224,7 +2790,9 @@ def _add_script_args(cls, parser):
         execution_group.add_argument(
             '-p', '--parallel',
             action='store_true',
-            help="Execute all operations in parallel.")
+            help="Execute all operations in parallel. This applies to "
+                 "bundling and groups. In combination this can lead to large "
+                 "resource requests.")
```

Comment:
Please remove the implicit warning. If folks want to run in parallel, then they want to run in parallel. It is not up to us to judge that or imply in any way that this might be bad practice.

Reply:
Will do. I just wanted to document that cavalier use of groups with bundling would waste a lot of compute resources.

Reply:
I understand the concern, but I don't think this is appropriate here. I think it would be much better if we develop some kind of resource check down the line where when you request more than 1% of the total resource on a given system you need to confirm that or whatever. But that's something we can worry about some other time.